tests/test_main.py

```diff
@@ -6,3 +6,4 @@ def test_health(app):
     response = app.get("/ping")
     assert response.status_code == 200
     assert response.json() == {"ping": "pong!"}
+    assert response.headers["x-total-time (ms)"]
```

Comment:
`x-total-time (ms)` or `x-total-time` @geospatial-jeff ?

Reply:
How about https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Server-Timing?

Reply:
```
response.headers["Server-Timing"]
>>> "total;dur={}"
```

I guess it will make sense to also follow that for our times in the `tile` endpoint


```
response.headers["Server-Timing"]
>>> "read;dur={}, postprocess;dur={}, format;dur={}, total;dur={}"
```

Reply:
Example for mosaic: `mosaicread;dur=0.83, read;dur=359.73, postprocess;dur=0.01, format;dur=125.54, total;dur=504.19`